SportsGoods.Core/Interfaces/IApplicationDbContext.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Microsoft.EntityFrameworkCore;
+using SportsGoods.Core.Models;
+
+namespace SportsGoods.Core.Interfaces
+{
+    public interface IApplicationDbContext
+    {
+        DbSet<Product> Products { get; }
+        //Task<int> SaveChangesAsync();
```

Comment:
if something is commented, then you don't need it, therefore no need to be in the PR

Reply:
Done (removed)